.changeset/sweet-squids-design.md

```diff
@@ -0,0 +1,6 @@
+---
+"@marigold/docs": patch
+"@marigold/theme-docs": patch
+---
+
+docs: make `code` in `<SectionMessage>` look nicer
```

Comment:
These are the patch notes our users see now in the whats new page ( I know before they also could read it) -  so I would maybe write more whats exactly looks nicer?